examples/s/d/nn/xcase/realization-0/iter-0/rms/bin/export_a_surface.py

```diff
@@ -39,6 +42,25 @@ def main():
     fname = ed.export(surf, name="all")  # note that 'name' here will be used
     print(f"File name is {fname}")
 
+    print("Export a depth map...")
+    surf = xtgeo.surface_from_file(FILES[1])
+    print(f"Average value of map is {surf.values.mean()}")
+
+    ed = dataio.ExportData(
+        config=CFG,
+        name="TopVolantis",  # will be translated to strat column name
+        unit="m",
+        vertical_domain={"depth": "msl"},
+        content="depth",
+        timedata=None,
+        is_prediction=True,
+        is_observation=False,
+        tagname="ds_extract_geogrid",
+        workflow="rms property model",
+    )
```

Comment:
Workflow for depth is "rms structural model"

In this routine, make a loop in main and separate the export to a more generic function, where a few keys are different between a property and a depth (the point is avoid too much duplicate code)

Reply:
> In this routine, make a loop in main and separate the export to a more generic function, where a few keys are different between a property and a depth (the point is avoid too much duplicate code)

Will involve quite a lot of config on the outside, and first attempts did not end up very readable compared to current. Problem is that this example is not really representative for actual usage. Normally, these would be in separate scripts, and each be >1 surface. Then it would make sense to loop. Can keep this open for discussion.